### PR TITLE
FEATURE: Allow user to interact with comments within post stream.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

**Review by pmusaraj — APPROVED** [review]
I pulled this locally, and it works very well. I very much like adding the comment in the post stream, and I think it makes lots of sense for the form for comments to be stripped down to a very simple textarea. 

There are a few rough edges: 

- after sending a comment, I see it twice (it's fixed on refresh) 
image
- We can remove the CSS width for the component container in https://github.com/discourse/discourse-question-answer/blob/main/assets/stylesheets/desktop/question-answer.scss#L9 -- all that does is cause issues on smaller screen sizes
- not sure we need comments on the OP, maybe we disable that button initially?

File: assets/javascripts/discourse/initializers/qa-edits.js.es6 [hunk 1]
```diff
@@ -97,46 +76,24 @@ function initPlugin(api) {
...
       }
 
-      result.push(helper.h("div.qa-comment-link", commentLinks));
+      if (helper.widget.attrs.canCreatePost) {
+        result.push(
+          helper.attach("qa-comment-menu", {
+            id: post.id,
+            moreCommentCount: post.comments_count - postCommentsLength,
+            lastPostNumber:
+              post.comments[post.comments.length - 1]?.post_number,
```
Comment (pmusaraj) [inline assets/javascripts/discourse/initializers/qa-edits.js.es6:93]: When I post an answer locally, I get a JS error pointing to this line: 

`Uncaught TypeError: Cannot read properties of undefined (reading 'length')`

Comment (tgxworld) [reply assets/javascripts/discourse/initializers/qa-edits.js.es6]: Thank you for catching this, I've fixed it.

**Comment by tgxworld** [comment]
Thank you for reviewing @pmusaraj :+1:

**Pull request merged**